src/webrtc/stats/statsReport.ts

```diff
@@ -79,4 +79,8 @@ export interface SummaryStatsReport {
     maxPacketLoss: number;
     percentageConcealedAudio: number;
     peerConnections: number;
+    roomStateExpectedPeerConnections: number;
+    missingPeerConnections: number;
+    percentageEstablishedPeerConnections: number;
+    // Todo: Decide if we want an index (or a timestamp) of this report in relation to the group call, to help differenciate when issues occur and ignore/track initial connection delays.
```

Comment:
I think the names are interpretations of the actual statistics. I would name the values according to their real meaning and do the interpretation in Posthog. And I would also add the joined users in the room. Because it is unusual for a user to take part in a conference with several devices. If it happens too often, we can use this to identify a caching issues.
```typescript
usersInRoom: number;
devicesInRoom: number;
diffDevicesToPeerConnections: number;
ratioPeerConnectionToDevices : number;
```

Reply:
To make the devices more comparable with the peerConnections i only count the devices without the "local" device. I changed it to `oppDevicesInCall` and `oppUsersInCall`.